pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -785,6 +817,22 @@ func (asw *actualStateOfWorld) PodExistsInVolume(podName volumetypes.UniquePodNa
 		return false, "", newVolumeNotAttachedError(volumeName)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.SELinuxMountReadWriteOncePod) {
+		if volumeObj.seLinuxMountContext != nil {
+			// The volume is mounted, check its SELinux context mount option
+			if *volumeObj.seLinuxMountContext != seLinuxLabel {
+				fullErr := newSELinuxMountMismatchError(volumeName)
+				if util.IsRWOP(volumeObj.spec) {
+					return false, volumeObj.devicePath, fullErr
+				} else {
```

Comment:
There is no metric to bump. And if there was one, it would be incremented every resync interval (100 ms), so it would not be that useful.
I am actually removing the `else` part with log, it would be logged really often and there already is one in `reconciler.mountOrAttachVolumes`.